site/content/en/docs/Reference/fleet.md

```diff
@@ -58,11 +58,11 @@ spec:
   # Which gameservers in the Fleet are most important to keep around - impacts scale down logic.
   # priorities:
   # - type: Counter # Sort by a “Counter”
-  #   key: player # The name of the Counter. No impact if no GameServer found.
-  #   order: Descending # Default is "Ascending" so smaller capacity will be removed first on down scaling.
+  #   key: rooms # The name of the Counter. No impact if no GameServer found.
```

Comment:
Ah good catch! Thank you! Got that in the [examples.yaml](https://github.com/googleforgames/agones/blob/release-1.40.0/examples/fleet.yaml), but not here.

https://github.com/googleforgames/agones/blob/main/examples/fleet.yaml has been updated with the move to beta -- but can we make the descriptions match across both (particularly the "available capacity" addition ?

Reply:
Thanks for the feedback! Changed in 1716e091d3f2804841d269200a6e31b01aea7ddb.

Added the Beta information and made the descriptions in fleet.yaml and fleet.md more consistent.